src/IonEventStream.ts

```diff
@@ -154,7 +158,7 @@ export class IonEventStream {
         let currentContainerIndex : number[] = [];
         while(true) {
             if(this.reader.isNull()){
-                this.eventStream.push(this.eventFactory.makeEvent(IonEventType.SCALAR, tid, this.reader.fieldName(), this.reader.depth(), this.reader.annotations(), true, this.reader.value()));
+                this.eventStream.push(this.eventFactory.makeEvent(IonEventType.SCALAR, tid, this.reader.fieldName(), this.reader.depth(), this.reader.annotations(), true, null));
```

Comment:
Can `reader` be on a container type here (i.e. `null.list`) and if so is it valid to make an event with `IonEventType.SCALAR` for container types?

Reply:
null containers are scalars

Reply:
According to what, exactly?

Reply:
because I designed it that way.

Reply:
Ok, well, the rest of the changes this PR are acceptable, but calling a `null.list` a scalar value seems wrong to me.  Whether or not a value is a scalar possibly should only be determined by its type, not its type and nulliness.  

I might be being pedantic here, but if is the event stream that is fed into to the compliance test tool, I'd suspect that you'd have to do whatever it expects, and I would be surprised if it considered null containers to be scalar.  

I'm approving this for now in the interest of moving forward.  

Besides, if my hunch is correct you'll have to fix it at a later date anyhow.